packages/slider/src/ux-slider.html

```diff
@@ -2,6 +2,7 @@
   <require from="./ux-slider.css"></require>
   <div class="ux-slider__container" 
        class.bind="disabled ? 'ux-slider__container--disabled' : 'ux-slider__container--enabled'"
+       touchstart.trigger="onTrackMouseDown($event)"
```

Comment:
With this, we won't have the focused effect on mobile, but it will probably make the component function properly. Pretty good fix I feel, though I think we may still need to stay aligned to general focused feedback of the UI on mobile

Reply:
I confirm that mobile now works properly (didn't work at all before).

Regarding the focused effect, what is the action that should lead to a focused state on mobile? My understanding is that

* when the touchstart occurs it already means that the use is interacting with the component and is rather active than focused
* focus is rather triggered by <TAB> like behavior such as clicking the "next" button when filling out a form...

Or do you think we should have a lifecycle such as

* touchstart => focus
* touchmove => active

Your thoughts ?

Reply:
we may not want to define focus/active states our own, so for the principle, it should be aligned to native (well browsers may do it differently)
If this is hard to defined, we can hold it for now and come back later to tackle it systematically across components I think (with some specs)

Reply:
Okey, if I understand correctly we can leave it as this for now and work on specs before to tackle this for all components. If someone knows how to do the specs part I think I'm happy to help to work on this later... I might struggle with the spec part though :-(